ElasticSynchronizer/Workers/DeleteProjectWorker.cs

```diff
@@ -46,7 +46,7 @@ public DeleteProjectWorker(ILogger<DeleteProjectWorker> logger, Config config, R
             this.restClient = restClient;
         }
 
-        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+        protected override Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
Where is this used?

(Just for my own information)
Will this actually run async if it's called with an await keyword?

Reply:
This is used to pickup registered Tasks from RabbitMQ. This class is reponsible to delete a project from the Elastic dataset.

The method was lacking await operators and this resulted in the method running synchronously and the IDE throwing warnings. I removed the async modifiers as they were useless anyways. Maybe should consider supressing the warnings here..